.changeset/large-zebras-exercise.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/modal': patch
+'@leafygreen-ui/popover': patch
+---
+
+Make react-transition-group an external dependency of the build
```

Comment:
This should be moved to a peer dependency then in the package.jsons for both these packages.

Reply:
👍